test/k8s-local-cluster-test/provision-eks-sqs-infra

```diff
@@ -0,0 +1,386 @@
+#!/bin/bash
+
+set -euo pipefail
+
+SCRIPTPATH="$( cd "$(dirname "$0")" ; pwd -P )"
+export AWS_PAGER=""
+export AWS_REGION=""
+ENDPOINT_URL="" #"--endpoint-url=http://localhost:4566" <- This is for localstack
+EXTRA_AWS_PARAMS=""
+DELETE=0
+FLEET=0
+ACCOUNT_ID=""
+CLUSTER_NAME=""
+SQS_QUEUE_NAME="MyK8sTermQueue"
+EVENTBRIDGE_ASG_RULE_NAME="MyK8sASGTermRule"
+EVENTBRIDGE_SPOT_RULE_NAME="MyK8sSpotTermRule"
+EVENTBRIDGE_EC2_STATE_CHANGE_RULE_NAME="MyK8sEC2StateChangeRule"
+LIFECYCLE_HOOK_NAME="my-k8s-term-hook"
+IAM_POLICY="term-event-policy"
+ROLE_NAME="term-event-role"
+POLICY_NAME="cth"
+RAND_ID=$(uuidgen | cut -d'-' -f1 | tr '[:upper:]' '[:lower:]')
+VERSION="v1.0.0-${RAND_ID}"
+SERVICE_ACCOUNT_NAME="aws-cluster-termination-handler-aws-node-termination-handler"
```

Comment:
i'm seeing `cluster-termination-handler` throughout this file-- will *cluster-termination* be referenced in the docs somewhere? could be confusing to see the name switch without context

Reply:
👍